cognite/neat/_graph/loaders/_rdf2dms.py

```diff
@@ -217,9 +218,11 @@ def _create_view_iterations(self) -> tuple[list[_ViewIterator], IssueList]:
         def sort_by_instance_type(id_: dm.ViewId) -> int:
             if id_ not in views_by_id:
                 return 0
-            return {"node": 1, "all": 2, "edge": 3}.get(views_by_id[id_].used_for, 0)
+            return {"node": 1, "all": 1, "edge": 3}.get(views_by_id[id_].used_for, 0)
 
-        ordered_view_ids = sorted(iterations_by_view_id.keys(), key=sort_by_instance_type)
+        ordered_view_ids = SchemaAPI.get_view_order_by_direct_relation_constraints(views)
```

Comment:
Any chance the views list here has no values?

Reply:
If the model is not deployed it would be empty. Will add a check for it.